diff --git a/example/blend.py b/example/blend.py
index 03b31b5..c89ef02 100644
--- a/example/blend.py
+++ b/example/blend.py
@@ -83,6 +83,7 @@ for sample_idx in range(1,11):
     sampling_params = SamplingParams(temperature=0, max_tokens=10)
     cache_fuse_metadata["check"] = True
     cache_fuse_metadata['collect'] = False
+    cache_fuse_metadata["suffix_len"] = len(q_ids + s_end)
     output = llm.generate([input_prompt], sampling_params)
     print(f"Cached generation: {output[0].outputs[0].text}")
     print(f"TTFT with cache: {output[0].metrics.first_token_time-output[0].metrics.first_scheduled_time}")
